ocaml/xapi/extauth_plugin_ADwinbind.ml

```diff
@@ -1083,6 +1084,105 @@ module ClosestKdc = struct
       Xapi_periodic_scheduler.remove_from_queue periodic_update_task_name
 end
 
+module RotateMachinePassword = struct
+  let task_name = "Rotating machine password"
+
+  let tmp_krb5_conf =
+    Printf.sprintf "%s/pwrotate.krb5" (Filename.get_temp_dir_name ())
```

Comment:
Is this risking a collision? This is global name - should this name not be unique for this invocation?

Reply:
This file does not need to be unique as
1. this is not parallel task
2. it can be simply overrided.